keps/sig-node/1769-memory-manager/README.md

```diff
@@ -778,3 +780,24 @@ The Memory Manager sets and enforces cgroup memory limit for ("on behalf of") a
 [hugepage-issue]: https://github.com/kubernetes/kubernetes/issues/80716
 [memory-issue]: https://github.com/kubernetes/kubernetes/issues/81009
 
+### Windows considerations
+
+Numa nodes can not be directly assigned or guaranteed via the Windows API.  Another limitation of the Windows API's is that [PROC_THREAD_ATTRIBUTE_PREFERRED_NODE](https://learn.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-updateprocthreadattribute) does not support setting this for the Job object (i.e. Container) and only supports setting a single Numa Node.
+The `PROC_THREAD_ATTRIBUTE_PREFERRED_NODE` api works by assigning workloads to the Numa Node via CPU affinity.  The API finds all processors associated with the Numa node then applies the CPU affinity to those processors which results in the memory from the Numa node being used. 
+In order to support multiple numa nodes and be able to apply the Numa affinity to job objects the container runtime will be expected to mimic
+the behavior of [PROC_THREAD_ATTRIBUTE_PREFERRED_NODE](https://learn.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-updateprocthreadattribute) 
+by finding the associated CPU's for the Numa Nodes that are passed via the Cri API and setting the preferred affinity for the job object. 
```

Comment:
Do we know that setting `PROC_THREAD_ATTRIBUTE_PREFERRED_NODE` is exactly equivalent to setting the job object's affinity to the CPUs on that node? Or could there be underlying differences?

Reply:
I've updated the wording here.  We don't know for certain if there are underlying differences, but will use the same general idea to work around the single Numa node limitation of this API.